src/test-helpers/exportWithoutRoot.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Remove root, de-indent (trim), and append newline to make tests cleaner.
+ */
```

Comment:
Unless you are using this function across multiple files, it should be defined in the module it is being used. There's no point adding to the surface area of the internal API if it is only used in one module.

Reply:
Yes, this is being used by [some tests](https://github.com/cybersemics/em/blob/2678137e03364e88b846d80096903279be79bc11/src/util/__tests__/importHtml.js#L53). Moreover, we can use it in other tests where we're using `exportContext`, to get rid of the `ROOT_TOKEN` from our tests' assertions, which is quite redundant.

Reply:
That sounds good. Please replace the duplicate code in other files with a reference to `exportWithoutRoot`.

Reply:
Sure. Just to be clear, are you referring to [this file](https://github.com/cybersemics/em/blob/2678137e03364e88b846d80096903279be79bc11/src/util/__tests__/importHtml.js#L53), or all the possible use-cases of this function that are currently using `exportContext`?

Reply:
Just duplicate functions that can be removed, like in importHtml, that look like this:

```js
  const exportedWithoutRoot = exported.slice(exported.indexOf('\n'))
    .split('\n')
    .map(line => line.slice(2))
    .join('\n')
    + '\n'
```